src/shared/components/Header/component.tsx

```diff
@@ -3,37 +3,40 @@ import { Container, Row, Col } from 'react-bootstrap';
 
 import { LogoBlackIcon } from 'shared/icons';
 
-import { MenuListItem } from './types';
 import Navigation from './Navigation';
 import SearchBar from './SearchBar';
+import Menu from './Menu';
 
-import { HeaderWrapper, LogoWrapper } from './styles';
+import { HeaderWrapper, LogoWrapper, HeaderElementsWrapper } from './styles';
 
 const Header: React.FC = () => {
-  const menuList: MenuListItem[] = [
-    { name: 'Знайти проекти', link: '#' },
-    { name: 'Додати проект', link: '/link2' },
-  ];
   const [isOpenSearch, setIsOpenSearch] = useState(false);
+  const [isOpenMenu, setIsOpenMenu] = useState(false);
   const onToggleHandler = (type: string): void => {
-    if (type === 'search') {
-      setIsOpenSearch(!isOpenSearch);
+    switch (type) {
+      case 'search':
```

Comment:
let's be more consistent - please put 'search' and 'menu' into some `enum` and use it where needed.

Reply:
Done!